net-p2p/ncdc/ncdc-1.22.1-r1.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
```

Comment:
Please look if you can use EAPI=8 and then revbump it if yes

Reply:
Wouldn't it require to drop stable keywords then? Why not start with latest version then?

Reply:
Well, it should be the best thing to do